google/cloud/bigquery_storage_v1/reader.py

```diff
@@ -149,15 +156,43 @@ def __iter__(self):
                 )
                 if not resumable_error:
                     raise
+            except google.api_core.exceptions.ResourceExhausted as exc:
+                # ResourceExhausted errors are only retried if a valid
+                # RetryInfo is provided with the error.
+                resumable_error = False
+                # ResourceExhausted doesn't seem to have
```

Comment:
This is likely because of the `google-api-core` version. Please add a `TODO:` to remove this logic when we require `google-api-core >= 2.2.0`.

https://github.com/googleapis/python-api-core/blob/main/CHANGELOG.md#220-2021-10-25

Reply:
Done